lib/src/revset.rs

```diff
@@ -572,9 +580,15 @@ static BUILTIN_FUNCTION_MAP: Lazy<HashMap<&'static str, RevsetFunction>> = Lazy:
         Ok(heads.ancestors_range(generation))
     });
     map.insert("descendants", |function, context| {
-        let [arg] = function.expect_exact_arguments()?;
-        let expression = lower_expression(arg, context)?;
-        Ok(expression.descendants())
+        let ([heads_arg], [depth_opt_arg]) = function.expect_arguments()?;
+        let heads = lower_expression(heads_arg, context)?;
```

Comment:
nit: it's not `heads` but `roots`.

Reply:
Ah, copied from ancestors without fully understanding. Thanks.